deploy/layered-sre-authorization/01-layered-cs-sre-admin-cluster.ClusterRole.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: layered-cs-sre-admin-cluster
+rules:
+# CS SRE can get, list, and watch all resources cluster wide
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Why cluster-scope get/list/watch?  My concern with this is it enables access to all secrets in the cluster and I'd just like to check that this is actually needed.

Reply:
Discussed on slack.  Basically it's there to enable access to the aws creds in case it's needed.  Discussed with @alanmoran about this being a break-glass thing, CS SRE could elevate privileges and get access.  Making elevating required means some thought has to go into the actions that are highly privileged.

Reply:
```suggestion
```